### PR TITLE
Bump kvo and tco

### PR DESCRIPTION
**Pull request by yifan-gu**
Bump kvo and tco

### CODE REVIEW

**Comment by yifan-gu** [comment]
This is a placeholder PR until kvo makes a release.

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
Hmm, upgrade test failed because the awsutil is removed in https://github.com/coreos/tectonic-installer/commit/c7978433db3b10605ee995c4c10e31e5c1e02fb9

**Comment by Quentin-M** [comment]
@yifan-gu TLS uploads will fail because of certificate SAN issue.

**Comment by yifan-gu** [comment]
@Quentin-M OK, then let me see if I can run aws cli in a container.

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

File: modules/update-payload/upload-payload.sh [hunk 1]
```diff
@@ -39,6 +39,8 @@ if [[ ${UPDATESERVICECTL} == "" ]]; then
     exit 1
 fi
 
+AWSCLI=${AWSCLI:-"aws"}
```
Comment (squat) [inline modules/update-payload/upload-payload.sh]: this is fine. are you making this variable so you can rkt run awscli optionally?

Comment (yifan-gu) [inline modules/update-payload/upload-payload.sh]: @squat Yes, in upgrade test's jenkins job [here](https://github.com/coreos-inc/tectonic-upgrade-test/blob/master/scripts/jenkins.sh#L219)

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Comment by yifan-gu** [comment]
Tests are passing happily, let me run one more time and then merge.

**Comment by yifan-gu** [comment]
test upgrade from 1.6.4-tectonic.1

**Pull request merged**